chain/cmd/bandsv/main.go

```diff
@@ -18,7 +30,130 @@ type OracleRequestResp struct {
 	Params    []byte `json:"params"`
 }
 
+type TxResult struct {
```

Comment:
Can't we reuse data structure from Tendermint, instead of re-defining one ourselves?